### DISCUSSION TITLE
gtlf without normals should use flat normals

### DISCUSSION THREAD

**Discussion by AMajesticSeaFlapFlap** [cables-gl/cables_docs#239] [! Welcome - readme first!]
According to the gltf 2.0 spec https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html :

> When normals are not specified, client implementations MUST calculate flat normals and the provided tangents (if present) MUST be ignored.

Cables, however, seems to use averaged normals if none are provided.

**Comment by steam0r** [comment]
we moved this to our backlog/ideas for a new version of gltf in cables
